src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs

```diff
@@ -345,13 +323,32 @@ protected virtual object CreateModel(ModelBindingContext bindingContext)
             // application developer should know that this was an invalid type to try to bind to.
             if (_modelCreator == null)
             {
+                // The following check causes the ComplexTypeModelBinder to NOT participate in binding structs as 
```

Comment:
Users override `CreateModel` when the need to take control of how the model is instantiated. Our limitations are not relevant unless they **do** call the `base.CreateModel()`.